Rakefile

```diff
@@ -68,7 +84,20 @@ task :chdir_pkg => [:package_dirs] do
 end
 
 task :deb => [:chdir_pkg, :propsd_source] do
-  sh "fpm --deb-no-default-config-files --deb-recommends nodejs -s dir -t deb -n \"#{name}\" -v #{version} opt/"
+  command = [
+    'fpm',
+    '--deb-no-default-config-files',
+    "--depends \"nodejs = #{target_version}\"",
```

Comment:
Is there a way to make this fuzzy? Right now, our target version is 4.3.2, but I'm pretty sure stuff would work on 4.4.0 as well. Would making this be `>= 4.x.x` do the right thing?